src/helpers/evm/evm.ts

```diff
@@ -30,8 +29,6 @@ export async function getProvider(
       return dwindow.BinanceChain;
     case "metamask":
       return dwindow.ethereum;
-    case "web3auth-torus":
-      return web3Auth.provider as InjectedProvider;
```

Comment:
Remove `REACT_APP_WEB3AUTH_CLIENT_ID` from *.env.example* and related constants in:
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9/src/constants/env.ts#L36-L38
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/RC-v1.9/amplify.yml#L24

Reply:
fixed